.pre-commit-config.yaml

```diff
@@ -30,6 +30,12 @@ repos:
     hooks:
       - id: pyupgrade
         args: ["--py3-plus"]
+  - repo: local
+    hooks:
+      - id: check-docstrings
+        name: check-docstrings
+        entry: scripts/check_docstrings
+        language: script
```

Comment:
This is a great idea.
I always forget that we can create `local` hooks with `pre-commit`